apps/server/src/modules/server/server.config.ts

```diff
@@ -10,6 +10,7 @@ export enum NodeEnvType {
 }
 
 interface IFeatureConfig {
+	FEATURE_JWT_EXTENDED_TIMEOUT_ENABLED: boolean;
```

Comment:
The possibility to use FEATURE_JWT_EXTENDED_TIMEOUT_ENABLED is not came from the server. It is directly connected to the authentication modul. If the module is deleted, replaced, move out from server it should be easy to identify which envirements also can be removed.

For this the modules can be define the possible interfaces and on this place the interfaces are only connected. 
I am not sure why IFeatureConfig is on this place. But for you env please define it first in module.

Reply:
The feature flag is unnecessary and was removed.